[2025-05-08T18:52:02.410+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-08T18:52:02.449+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: google_maps_reviews.run_dbt_models scheduled__2025-05-07T00:00:00+00:00 [queued]>
[2025-05-08T18:52:02.470+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: google_maps_reviews.run_dbt_models scheduled__2025-05-07T00:00:00+00:00 [queued]>
[2025-05-08T18:52:02.471+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-08T18:52:02.498+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): run_dbt_models> on 2025-05-07 00:00:00+00:00
[2025-05-08T18:52:02.517+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=252) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-08T18:52:02.521+0000] {standard_task_runner.py:72} INFO - Started process 267 to run task
[2025-05-08T18:52:02.521+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'google_maps_reviews', 'run_dbt_models', 'scheduled__2025-05-07T00:00:00+00:00', '--job-id', '799', '--raw', '--subdir', 'DAGS_FOLDER/***dag.py', '--cfg-path', '/tmp/tmp39vt5i_t']
[2025-05-08T18:52:02.525+0000] {standard_task_runner.py:105} INFO - Job 799: Subtask run_dbt_models
[2025-05-08T18:52:02.627+0000] {task_command.py:467} INFO - Running <TaskInstance: google_maps_reviews.run_dbt_models scheduled__2025-05-07T00:00:00+00:00 [running]> on host 9ce8fafe7ab9
[2025-05-08T18:52:02.798+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='tonemail@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='google_maps_reviews' AIRFLOW_CTX_TASK_ID='run_dbt_models' AIRFLOW_CTX_EXECUTION_DATE='2025-05-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-07T00:00:00+00:00'
[2025-05-08T18:52:02.801+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-08T18:52:02.802+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-08T18:52:02.802+0000] {logging_mixin.py:190} INFO - Current task name:run_dbt_models state:running start_date:2025-05-08 18:52:02.450869+00:00
[2025-05-08T18:52:02.803+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews and current dag run status:running
[2025-05-08T18:52:02.804+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-08T18:52:02.806+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-05-08T18:52:02.808+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt_project/bank_reviews_dbt && dbt run --profiles-dir .']
[2025-05-08T18:52:02.839+0000] {subprocess.py:99} INFO - Output:
[2025-05-08T18:52:08.799+0000] {subprocess.py:106} INFO - [0m18:52:08  Running with dbt=1.9.4
[2025-05-08T18:52:09.576+0000] {subprocess.py:106} INFO - [0m18:52:09  Registered adapter: postgres=1.9.0
[2025-05-08T18:52:10.978+0000] {subprocess.py:106} INFO - [0m18:52:10  Found 5 models, 12 data tests, 433 macros
[2025-05-08T18:52:10.986+0000] {subprocess.py:106} INFO - [0m18:52:10
[2025-05-08T18:52:10.989+0000] {subprocess.py:106} INFO - [0m18:52:10  Concurrency: 1 threads (target='dev')
[2025-05-08T18:52:10.991+0000] {subprocess.py:106} INFO - [0m18:52:10
[2025-05-08T18:52:11.260+0000] {subprocess.py:106} INFO - [0m18:52:11  1 of 5 START sql table model review.dim_bank ................................... [RUN]
[2025-05-08T18:52:11.691+0000] {subprocess.py:106} INFO - [0m18:52:11  1 of 5 OK created sql table model review.dim_bank .............................. [[32mSELECT 268[0m in 0.42s]
[2025-05-08T18:52:11.701+0000] {subprocess.py:106} INFO - [0m18:52:11  2 of 5 START sql table model review.dim_branch ................................. [RUN]
[2025-05-08T18:52:11.896+0000] {subprocess.py:106} INFO - [0m18:52:11  2 of 5 OK created sql table model review.dim_branch ............................ [[32mSELECT 268[0m in 0.19s]
[2025-05-08T18:52:11.902+0000] {subprocess.py:106} INFO - [0m18:52:11  3 of 5 START sql table model review.dim_location ............................... [RUN]
[2025-05-08T18:52:12.082+0000] {subprocess.py:106} INFO - [0m18:52:12  3 of 5 OK created sql table model review.dim_location .......................... [[32mSELECT 740[0m in 0.18s]
[2025-05-08T18:52:12.091+0000] {subprocess.py:106} INFO - [0m18:52:12  4 of 5 START sql table model review.dim_sentiment .............................. [RUN]
[2025-05-08T18:52:12.252+0000] {subprocess.py:106} INFO - [0m18:52:12  4 of 5 OK created sql table model review.dim_sentiment ......................... [[32mSELECT 3[0m in 0.16s]
[2025-05-08T18:52:12.257+0000] {subprocess.py:106} INFO - [0m18:52:12  5 of 5 START sql table model review.fact_reviews ............................... [RUN]
[2025-05-08T18:52:12.414+0000] {subprocess.py:106} INFO - [0m18:52:12  5 of 5 OK created sql table model review.fact_reviews .......................... [[32mSELECT 4416[0m in 0.15s]
[2025-05-08T18:52:12.449+0000] {subprocess.py:106} INFO - [0m18:52:12
[2025-05-08T18:52:12.451+0000] {subprocess.py:106} INFO - [0m18:52:12  Finished running 5 table models in 0 hours 0 minutes and 1.46 seconds (1.46s).
[2025-05-08T18:52:12.611+0000] {subprocess.py:106} INFO - [0m18:52:12
[2025-05-08T18:52:12.613+0000] {subprocess.py:106} INFO - [0m18:52:12  [32mCompleted successfully[0m
[2025-05-08T18:52:12.615+0000] {subprocess.py:106} INFO - [0m18:52:12
[2025-05-08T18:52:12.616+0000] {subprocess.py:106} INFO - [0m18:52:12  Done. PASS=5 WARN=0 ERROR=0 SKIP=0 TOTAL=5
[2025-05-08T18:52:14.857+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-05-08T18:52:14.907+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-08T18:52:14.908+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=google_maps_reviews, task_id=run_dbt_models, run_id=scheduled__2025-05-07T00:00:00+00:00, execution_date=20250507T000000, start_date=20250508T185202, end_date=20250508T185214
[2025-05-08T18:52:14.952+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-08T18:52:14.953+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-08T18:52:14.954+0000] {logging_mixin.py:190} INFO - Dag name:google_maps_reviews queued_at:2025-05-08 18:48:15.030563+00:00
[2025-05-08T18:52:14.955+0000] {logging_mixin.py:190} INFO - Task hostname:9ce8fafe7ab9 operator:BashOperator
[2025-05-08T18:52:15.005+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-08T18:52:15.041+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-08T18:52:15.048+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
